stage-sites.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+curl 'https://www.cardinalsites.acsitefactory.com/api/v2/stage' \
+  -X POST -H 'Content-Type: application/json' \
+  -d '{"to_env": "test",
+  "sites": [10386,9996,9951,10291,12791,9931,10236,10381,10436,9466,10441,10446,9956,10226,10426,10461,10676,16621,17387,9361,16136,19006,16661,9956,17676,14656,17281,16471,19456,8011,10711,16871,16901,16896,16881,16866,16891,16876,6886,16236,13831,16796,16871,20986,16866,16876,16896,16901,16891,19236,16881,19691,20491,10696,16611,21996,22001,17671,17666,16611,17091,20246,21976,17446,21736,21661,20391,22181,17151,21476,20241,17451,15721,20396,22416,1576],
```

Comment:
FYI: If any of those id's doesn't exist on prod, the request will error. so a possible future would be if any of these sites was retired and deleted, this will break.

Reply:
Yeah, there are definitely improvements to be made. This ideally would be in PHP or _something-other-than-bash_, for one, that would allow for better error handling.

Reply:
well that's basically what this pr does: https://github.com/SU-SWS/acsf-cardinalsites/pull/239